package.json

```diff
@@ -11,9 +11,15 @@
     "coverage": "nyc npm test",
     "coverage:html": "nyc npm test && nyc report --reporter=html",
     "coveralls": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
-    "docs": "./node_modules/.bin/jsdoc --verbose -d docs -t ./node_modules/ink-docstrap/template -R README.md index.js",
+    "docs": "documentation build index.js --shallow -f html -o docs --config documentation.yml",
```

Comment:
Why are you using the shallow flag here?

Reply:
I just wanted to document the index.js file, since that is the only API that is exposed.